src/frontend/components/ToggleButton/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { CheckBox, Grommet } from 'grommet';
+import { deepMerge, normalizeColor } from 'grommet/utils';
+import React from 'react';
+
+import { theme } from 'utils/theme/theme';
+
+const customToggleTheme = {
+  checkBox: {
+    border: {
+      color: normalizeColor('blue-active', theme),
+      width: '1px',
+    },
+    color: normalizeColor('bg-select', theme),
+    hover: {
+      border: {
+        color: 'none',
+      },
+    },
+    toggle: {
+      background: ({ checked }: { checked: boolean }) =>
+        checked ? normalizeColor('blue-active', theme) : 'transparent',
+      color: ({ checked }: { checked: boolean }) =>
+        checked
+          ? normalizeColor('bg-select', theme)
+          : normalizeColor('blue-active', theme),
+      knob: {
+        extend: `
+          top: 1px;
+          left: 2px;
+          height: 12px;
+          width: 12px;
+          `,
+      },
+      size: '40px',
+      extend: `
+          width: 32px;
+          height: 16px;
+          box-shadow: none;
+        `,
+    },
+  },
+};
+
+interface ToggleButton {
```

Comment:
What do you think about extending this interface from `CheckBoxExtendedProps` ? This allows to use all the props defined by the `CheckBox` grommet component

Reply:
yes, done